ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -636,4 +636,5 @@ FAN_SS_VSC_EPC_EHHRC_EHHRC3X:
   uses:
   - heat_recovery_water_isolation_valve_command_1
   - heat_recovery_water_isolation_valve_command_2
-  - heat_recovery_water_isolation_valve_command_3
\ No newline at end of file
+  - heat_recovery_water_isolation_valve_command_3
```

Comment:
please revert if just whitespace.

Reply:
It is just a white space. No modification has been done for this....